contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1209,15 +1209,15 @@ $ oc get endpoints --all-namespaces --template \
 # subscription-manager list
 ----
 
-* For snippets or sections of a file, use an ellipsis (`...`) to show that the file continues before and/or after the quoted block.
+* For snippets or sections of a file, use an ellipsis (`...` or `# ...` for YAML) to show that the file continues before and/or after the quoted block.
```

Comment:
@jboxman : ellipis without the hyphen has no context in YAML?

Reply:
I tried to remove the slash construction, but it's difficult.

Can we stick with `before or after`? If one were to use `#...` in their example, they would usually have line content either before or after this syntax.

Reply:
Doesn't matter to me. I kept the existing sentence and just added the YAML ellipsis. I think 'before or after' looks better than the slash and makes more sense. Will update.